cypress/e2e/orders/edifact-export-to-a-vendor.cy.js

```diff
@@ -0,0 +1,167 @@
+import DateTools from '../../support/utils/dateTools';
+
+describe('orders: export', () => {
+  
+  // Test will be failed with issue https://issues.folio.org/browse/STCOM-1120
+  
+  const order = { ...NewOrder.defaultOneTimeOrder };
+  const organization = {
+    ...NewOrganization.defaultUiOrganizations,
+    accounts: [
+      {
+        accountNo: getRandomPostfix(),
+        accountStatus: 'Active',
+        acqUnitIds: [],
+        appSystemNo: '',
+        description: 'Main library account',
+        libraryCode: 'COB',
+        libraryEdiCode: getRandomPostfix(),
+        name: 'TestAccout1',
+        notes: '',
+        paymentMethod: 'Cash',
+      },
+      {
+        accountNo: getRandomPostfix(),
+        accountStatus: 'Active',
+        acqUnitIds: [],
+        appSystemNo: '',
+        description: 'Main library account',
+        libraryCode: 'COB',
+        libraryEdiCode: getRandomPostfix(),
+        name: 'TestAccout2',
+        notes: '',
+        paymentMethod: 'Cash',
+      },
+    ]
+  };
+  const integrationName1 = `FirstIntegrationName${getRandomPostfix()}`;
+  const integrationName2 = `SecondIntegrationName${getRandomPostfix()}`;
+  const integartionDescription1 = 'Test Integation descripton1';
+  const integartionDescription2 = 'Test Integation descripton2';
+  const vendorEDICodeFor1Integration = getRandomPostfix();
+  const libraryEDICodeFor1Integration = getRandomPostfix();
+  const vendorEDICodeFor2Integration = getRandomPostfix();
+  const libraryEDICodeFor2Integration = getRandomPostfix();
+  let user;
+  let location;
+  let servicePointId;
+  let institutionId;
+  let campusId;
+  let libraryId;
+  let institutionName;
+  const UTCTime = DateTools.getUTCDateForScheduling();
+
+  before(() => {
+    cy.getAdminToken();
+
+    ServicePoints.getViaApi()
+        .then((servicePoint) => {
+            servicePointId = servicePoint[0].id;
+            Institutions.createViaApi(Institutions.getDefaultInstitutions())
+                .then(locinsts => {
+                    institutionId = locinsts.id;
+                    institutionName = locinsts.name;
+                    Campuses.createViaApi({ ...Campuses.getDefaultCampuse(), institutionId })
+                        .then(loccamps => {
+                            campusId = loccamps.id;
+                            Libraries.createViaApi({ ...Libraries.getDefaultLibrary(), campusId })
+                                .then(loclibs => {
+                                    libraryId = loclibs.id;
+                                    NewLocation.createViaApi(NewLocation.getDefaultLocation(servicePointId,institutionId,campusId,libraryId))
+                                        .then(locationResponse => {
+                                            location = locationResponse;
+                                        });
+                                });
+                        });
+                });
```

Comment:
we don't need to create a new method .
we have already the method that created location. campus, institution

for instance this is the piece of code you can apply in your situation
.defaultLocation = Location.getDefaultLocation(testData.servicePointS.id);
        Location.createViaApi(testData.defaultLocation);

Reply:
Fix it and remove all to one method